incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/view/MessageView.java

```diff
@@ -22,13 +22,20 @@
 public final class MessageView extends Resolvable<Message>
 {
     private final Message message;
+    private final String refKey;
 
     private MessageView(
         Map<String, Message> messages,
         Message message)
     {
         super(messages, "#/components/messages/(\\w+)");
         this.message = message.ref == null ? message : resolveRef(message.ref);
+        this.refKey = this.key;
+    }
+
+    public String refKey()
+    {
+        return refKey;
```

Comment:
Should this just be `return key;` since `refKey` is assigned from `this.key`?

Reply:
fixed